src/torchcodec/_core/AVIOFileLikeContext.h

```diff
@@ -17,7 +17,8 @@ namespace facebook::torchcodec {
 
 // Enables uers to pass in a Python file-like object. We then forward all read
 // and seek calls back up to the methods on the Python object.
-class AVIOFileLikeContext : public AVIOContextHolder {
+class __attribute__((visibility("hidden"))) AVIOFileLikeContext
+    : public AVIOContextHolder {
```

Comment:
I don't love that we need this again, but that seems to be the case. This is, I think, because now the `custom_ops.cpp` side needs to see it.